docs/docs/keymaps/behaviors/layers.md

```diff
@@ -42,7 +26,7 @@ again.
 Example:
 
 ```dts
-&mo LOWER
+&mo 3
 ```
 
```

Comment:
Can I suggest you add a e.g. `note` admonition box here, reminding that layer numbers correspond to the order they are defined under the keymap node, starting from zero? e.g. the `3` here would be the 4th node, etc.

Having the `#define`s earlier sort of illustrated this before.

Reply:
I added a new line to the admonition box just above here. Having just one highlighted section feels cleaner than two right next to each other to me. The linked page also have more info about layer numbers with examples. What do you think?

Reply:
I thought about the note box right after the first time it is used in an example, e.g. here after `&mo 3` but I guess they would still be pretty close. I think yours is fine, yeah.